src/Evolution/Initialization/Domain.hpp

```diff
@@ -91,7 +91,7 @@ struct Domain {
     ElementMap<Dim, Frame::Inertial> map{element_id,
                                          my_block.coordinate_map().get_clone()};
 
-    return std::make_tuple(add_nonexistent_to_databox<Domain, compute_tags>(
+    return std::make_tuple(merge_into_databox<Domain, compute_tags>(
```

Comment:
Can you make the `Action` template parameter a meta argument instead or move it after the simple and compute tags list template parameters, so that it doesn't break the nice `merge_into_databox<the_tags_to_merge>` semantics of reading this?

Reply:
In theory yes, it makes the code harder to use in my opinion. I'm very strongly against this suggestion.